packages/rocket-webhook-types/src/webhook-params.ts

```diff
@@ -9,13 +9,19 @@ export type WebhookAuthorizer = (currentUser?: UserEnvelope, request?: WebhookRe
 
 export type WebhookRoleAccess = 'all' | Array<Class<RoleInterface>> | WebhookAuthorizer
 
+export enum AllowedHttpMethods {
+  GET = 'GET',
+  POST = 'POST',
+}
+
 export interface WebhookParamsEvent {
   // @deprecated Use route parameter
   origin?: string
   route?: string
   handlerClass: WebhookHandlerClassInterface
   multiPartConfig?: MultiPartConfig
   authorize?: WebhookRoleAccess
+  allowedMethods?: Array<AllowedHttpMethods>
```

Comment:
with the change above, this property would be
```typescript
allowedMethods?: Array<AllowedHttpMethod>
```

Reply:
Changed in commit edffcb5.